package com.example.Aula19052026.dto.cliente;

public class ClienteResponseDTO {

    private String nome;
    private String email;


    public ClienteResponseDTO() {
    }

    public ClienteResponseDTO(String nome, String email) {
        this.nome = nome;
        this.email = email;
    }

    public String getNome() {
        return nome;
    }

    public void setNome(String nome) {
        this.nome = nome;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }
}
